packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1321,47 +1316,47 @@ HTTP/1.1 204 No Content
 }
 ```
 
-### Role Properties
+### Role properties
 
-The role model defines several **read-only** properties:
+The Role model defines several **read-only** properties:
 
 | Property         | Description                                             | DataType                                                                                                                                   | Nullable   | Unique   | Read Only |
 | :--------------- | :------------------------------------------------------ | :------------------------------------------------------------------------------------------------------------                              | :--------- | :------- | :-------- |
-| id               | Unique key for the role assignment                      | String                                                                                                                                     | FALSE      | TRUE     | TRUE      |
-| label            | Display name of role                                    | String                                                                                                                                     | FALSE      | FALSE    | TRUE      |
-| type             | Type of role                                            | `SUPER_ADMIN`, `ORG_ADMIN`, `APP_ADMIN`, `USER_ADMIN`, `HELP_DESK_ADMIN`, `READ_ONLY_ADMIN`, `MOBILE_ADMIN`, `API_ACCESS_MANAGEMENT_ADMIN`, `REPORT_ADMIN` | FALSE      | FALSE    | TRUE      |
-| status           | Status of role assignment                               | `ACTIVE`                                                                                                                                   | FALSE      | FALSE    | TRUE      |
+| _embedded        | Embedded resources related to the role assignment       | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06)                                                                             | TRUE       | FALSE    | TRUE      |
```

Comment:
same here I think we want to keep the order of properties as they appear in JSON response as opposed to alphabetical order.

Reply:
We do not want that.